### PR TITLE
add Google stuff

### PR DESCRIPTION
**Pull request by Yay295**
supposedly this might allow Google to add a little search box under openings.moe in Google's search results

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -107,6 +107,22 @@ function videoMIMEsubtype() {
...
+		<!-- Google Stuff -->
+		<meta name="google-site-verification" content="J8ZBk2wWSOvLneswCoGGHmrwRPBy9HbX7RHxAGFgz4E" />
+		<script type="application/ld+json">
+			{
+				"@context" : "http://schema.org",
+				"@type" : "WebSite",
+				"url" : "https://openings.moe/",
+				"potentialAction" : {
+					"@type" : "SearchAction",
+					"target" : "https://openings.moe/search/?s={search_term_string}",
```
Comment (rm-you) [inline index.php]: Did you forget to add something to your commit? I assume /search/index.php should exist as part of this PR?

**Comment by QuadPiece** [comment]
May I ask where the `<meta name="google-site-verification" content="J8ZBk2wWSOvLneswCoGGHmrwRPBy9HbX7RHxAGFgz4E" />` was obtained. Seems to include a key for verification.

**Comment by Yay295** [comment]
From Google. This doesn't seem to have worked anyway so it doesn't really matter if it's added or not.

**Comment by QuadPiece** [comment]
I was referring more to what it was linked to. Since I already have the site registered in Webmaster tools. But either way, if it's not functional then I'll just close it.

**Pull request closed without merging**